As3ToHaxe.hx

```diff
@@ -123,7 +126,15 @@ class As3ToHaxe
 		// package
 		var r = new EReg("package ([a-zA-Z\\.0-9-_]*)([ \n\r]*){", "gs");
 		r.match(s);
```

Comment:
I think this returns boolean value, which is `True` if RegEx is matched, it's better to use it instead of just skipping error. Thank you.